task_13/test/test.js

```diff
@@ -0,0 +1,45 @@
+const assert = require('chai').assert
+const {chef, Calculator} = require('../app')
+
+describe('Chef test', ()=>{
+    let cheff = chef
+    
+    it('check if the dish has a valid name', ()=>{
+        assert.isString(cheff.checkMenu(), 'string')
+    })
+
+    it('check for a dish in a menu', ()=>{
+        let dish = cheff.checkMenu()
+        assert.oneOf(dish, cheff.dishes)
+    })
+
+    it('make sure the cheff can feed more', ()=>{
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887248)
```suggestion
    it("make sure the cheff can feed more", ()=>{
```